taskboot/push.py

```diff
@@ -73,35 +73,41 @@ def heroku_release(target, args):
     # Load queue service
     queue = taskcluster.Queue(config.get_taskcluster_options())
 
-    # Get the list of matching artifacts as we should get only one
-    matching_artifacts = load_artifacts(args.task_id, queue, args.artifact_filter, args.exclude_filter)
-
-    # Push the Docker image
-    if len(matching_artifacts) == 0:
-        raise ValueError(f"No artifact found for {args.artifact_filter}")
-    elif len(matching_artifacts) > 1:
-        raise ValueError(f"More than one artifact found for {args.artifact_filter}: {matching_artifacts!r}")
-    else:
+    updates_payload = []
+
+    for artifact in args.artifacts:
+        heroku_dyno_type, artifact_path = artifact.split(":", 1)
```

Comment:
Please add an assert with a nice error message on the presence of `:` in the full artifact string. Otherwise the exception will be `ValueError: not enough values to unpack (expected 2, got 1)` which is not really helpful

Reply:
Will do